Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake_test.py

```diff
@@ -196,7 +196,7 @@ def test_query_logs_command_transform_results_1():
     cdl_records_xform = load_test_data('./test_data/test_query_logs_command_transform_results_xformed.json')
 
     class MockClient():
-        def query_loggings(self, query):
+        def query_loggings(self, query, page_number=None, page_size=None):
```

Comment:
What does this addition add to the test?
How does it test the new use cases you entered?

Reply:
The signature of the mock method had to be modified to match the original method